requirements/flake8.txt

```diff
@@ -20,6 +20,8 @@ mccabe==0.6.1 \
     --hash=sha256:ab8a6258860da4b6677da4bd2fe5dc2c659cff31b3ee4f7f5d64e79735b80d42 \
     --hash=sha256:dd8d182285a0fe56bace7f45b5e7d1a6ebcbf524e8f3bd87eb0f125271b8831f
 # pycodestyle is required by flake8
+# See https://gitlab.com/pycqa/flake8/issues/444 and related merge requests
+# for why this is pinned.
```

Comment:
https://gitlab.com/pycqa/flake8/issues/444 is closed though - is there something else it's waiting on?

Reply:
We're waiting for the actual release, yeah.